src/main/cli/settings.c

```diff
@@ -1131,7 +1131,11 @@ const clivalue_t valueTable[] = {
 
     { "angle_level_strength",       VAR_UINT8  | PROFILE_VALUE, .config.minmaxUnsigned = { 0, 200 }, PG_PID_PROFILE, offsetof(pidProfile_t, pid[PID_LEVEL].P) },
```

Comment:
Now's a good time to  use param_name values for all angle and horizon CLI parameters.  This frees up flash space if we are going to send names to blackbox or OSD.

Reply:
agreed. Once we're happy with the functionality and naming I'll convert them.